lib/rspec/matchers.rb

```diff
@@ -646,6 +646,7 @@ def include(*expected)
     alias_matcher :a_string_including,     :include
     alias_matcher :a_hash_including,       :include
     alias_matcher :including,              :include
+    alias_matcher :have_a_member,          :include
```

Comment:
> I don't want to add have_a_member to RSpec. Any user who wants it can add it with a one-liner.

No, problem at all. :smile: Will suggest it again. Will fix the example by using your suggestion.